frontend/src/Pages/AdminPage.js

```diff
@@ -28,14 +32,30 @@ export default function AdminPage() {
     return wrongAnswers.length
   }
 
+  const howMany = answers => {
+    const users = answers.map(answer => answer.userName)
+    console.log(users)
+    const number = [...new Set(users)]
+    console.log(number)
+    return number.length
+  }
```

Comment:
```suggestion
  const getNumberOfUsers = answers => {
    const username = answers.map(answer => answer.userName)
    const distinctUsers = new Set(users)
    return distinctUsers.size
  }
```

This is pure function an could be easily extracted to a service file ;)

Reply:
Of course I forgot some  console.logs :D Service file sounds good, maybe with the next refactor 👍